khushaal nandwani's submission for challenge

khushaal-nandwani opened the pull request:
Please read README2.md for concerns

doublederek commented:
Hi there, I've updated the test file to reflect the proper number output, please pull down the changes and make any updates your PR as needed. Leave a comment when you do and I'll run the test. Thanks!

khushaal-nandwani commented:
Hi, my older version of `translator.py` was already working. I had changed the `translator.test.py`. I have updated it to the newer one now. 
Thanks:)
